src/libs/ajax.js

```diff
@@ -1684,6 +1684,11 @@ const Dockstore = signal => ({
   getVersions: async ({ path, isTool }) => {
     const res = await fetchDockstore(dockstoreMethodPath({ path, isTool }), { signal })
     return res.json()
+  },
+
+  listTools: async (params = {}) => {
+    const response = await fetchDockstore(`api/ga4gh/v1/tools?${qs.stringify(params)}`)
+    return response.json()
   }
```

Comment:
I know this follows the existing pattern, but it is unnecessarily verbose, in my opinion. I'm not sure anyone has the bandwidth to put in a refactoring PR to address this universally, and in any case I'd like to start small, so I suggest:
```suggestion
  listTools: (params = {}) =>
    fetchDockstore(`api/ga4gh/v1/tools?${qs.stringify(params)}`).then(r => r.json())
```